NEWS.md

```diff
@@ -7,22 +7,19 @@ Note that each entry is kept to a minimum, see links for details.
 
 ## Language changes
 
-* String literals in files without a `frozen_string_literal` comment now behave
-  as if they were frozen. If they are mutated a deprecation warning is emited.
-  These warnings can be enabled with `-W:deprecated` or by setting `Warning[:deprecated] = true`.
-  To disable this change you can run Ruby with the `--disable-frozen-string-literal` command line
-  argument. [[Feature #20205]]
+*String literals in files without a frozen_string_literal comment now behave as if they were frozen.
+If they are mutated a deprecation warning is emitted. These warnings can be enabled with -W:deprecated or by setting Warning[:deprecated] = true. 
+To disable this change, you can run Ruby with the --disable-frozen-string-literal command line argument. [Feature #20205]
 
-* `it` is added to reference a block parameter. [[Feature #18980]]
+*A new syntax **'it': value is added to reference a block parameter. [Feature #18980]
```

Comment:
I can't get your intention.
What does `value` mean here?

Reply:
it make it easy to read as it show that 'it' is a syntex

Reply:
It is the part "A new syntax", right?
I'm not sure where/how the sentence is separated.

`A new syntax **'it'` and `value is added ...`?
Or `A new syntax` means `**'it': value`?
The latter does not make sense obviously, but then what does `value` mean in the former?